SS14.Loader/Program.cs

```diff
@@ -9,15 +9,29 @@
 using NSec.Cryptography;
 using Robust.LoaderApi;
 
+
 namespace SS14.Loader;
 
+internal enum ExitCode
```

Comment:
Instead of casting every time I believe you can just make the underlying type int

```suggestion
internal enum ExitCode : int
```